app/scripts/modules/core/task/modal/reason.directive.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.task.reason.directive', [])
+  .directive('taskReason', function () {
+    return {
+      restrict: 'E',
+      bindToController: {
+        command: '=',
+      },
+      controller: angular.noop,
+      controllerAs: 'vm',
+      templateUrl: require('./reason.directive.html')
```

Comment:
I think you want to include a scope declaration here; otherwise, it will inherit the parent scope (not a big deal, since the changes are so isolated, but a good practice, anyway). Just need to add: `scope: {}` here.